File: week2/scores.c
#include <stdio.h>
#include <cs50.h>

// A program that prints out the average value of specified numbers

const int TOTAL = 3;

//Prototype
float average(int length, int array[]);

int main(void)
{
    int scores[TOTAL];

    // Prompt user to insert scores
    for (int i = 0; i < TOTAL; i++)
    {
        scores[i] = get_int("Score: ");
    }
    printf("Average is: %f.\n", average(TOTAL, scores));
}

float average(int length, int array[])
{
    int sum = 0;

    // Sum scores in array
    for (int i = 0; i < length; i++)
    {
        sum = sum + array[i];
    }
    // Calculate average
    return sum / length;
}

File: week6/scores.py
# A program that calculates the average of scores

from cs50 import get_int

# Empty list
scores = []

# Number of scores
length = get_int("How many scores do you want to insert? ")

# Append scores in scores list
for i in range(length):
    n = get_int("Score: ")
    scores.append(n)

# Calculate average
average = sum(scores) / length

# Print average
print(f"Average: {average}")